tests/ui/cast/trait-object-cast-segfault-4333.rs

```diff
@@ -7,3 +7,5 @@ pub fn main() {
     let stdout = &mut io::stdout() as &mut dyn io::Write;
     stdout.write(b"Hello!");
 }
+
+// https://github.com/rust-lang/rust/issues/4333
```

Comment:
Remark: hm, I think I would still prefer having these comments at the top of the test, and keep two commits {move, edits}. It's fine for this PR, but I do still think it's nicer to have the doc comments / backlinks on start of file.